.esbuild/util.cjs

```diff
@@ -26,12 +26,14 @@ const buildOptions = (override = {}) => {
   };
 };
 
-const getOutFiles = (extension) => {
+const getEntryPoints = (extension) => {
   return {
     [`mermaid${extension}`]: 'src/mermaid.ts',
     [`diagramAPI${extension}`]: 'src/diagram-api/diagramAPI.ts',
```

Comment:
Why is this `diagramAPI` separate?

(It's in commit https://github.com/mermaid-js/mermaid/commit/a87abc00c65de91ed417e25c5f81973673f21153, which isn't in this PR, so this might not be the best place to discuss this.)

It seems like it just makes the `dist/` folder twice as big (about 26 MB larger!).

Reply:
That was requested by @knsv for splitting the diagrams out. We are still discussing the implementation details.
Let's continue the discussion in Esbuild 👍.